### ISSUE TITLE
[Feature Request]: Add asset serial in license seats view

### ISSUE THREAD

**Issue by cuuush** [grokability/snipe-it#11686]
### Is your feature request related to a problem? Please describe.

Adding a column for asset serial in the license seats view would help those who use licenses to represent hardware support contracts (Juniper J-Care, Cisco Smart Net). The serial number of the asset is useful to create reports (using the table export feature) of which serial numbers are associated with the support contracts.

### Describe the solution you'd like

add an asset serial column to the license seats view

**Comment by snipe** [comment]
Duplicate of #4729

**Comment by cuuush** [comment]
I don't believe these are duplicate. I'm not asking for multiple product keys for a license, I just want to view an asset's serial when looking at the list of seats. So when checking a seat out to an asset, instead of just being able to see the name, I'd like to display the name and the serial of the asset.

we currently accomplish this by modifying LicensePresenter and LicenseSeatsTransformer to display asset serial in the columns drop down.

**Comment by snipe** [comment]
Ah, got it

**Comment by snipe** [comment]
Thanks for clarifying!

**Comment by snipe** [comment]
The problem here (and always has been) that license seats can be checked out to both users and assets, so adding a serial column when a particular company's workflow is that they only assigned to users would be a confusing UI.
